test/parallel/test-os.js

```diff
@@ -191,3 +181,14 @@ is.string(pwd.username);
 assert.ok(pwd.homedir.includes(path.sep));
 assert.strictEqual(pwd.username, pwdBuf.username.toString('utf8'));
 assert.strictEqual(pwd.homedir, pwdBuf.homedir.toString('utf8'));
+
+// Test that the Symbol.toPrimitive functions work correctly
+[
+  [assert.strictEqual, `${os.hostname}`, os.hostname()],
+  [assert.strictEqual, `${os.homedir}`, os.homedir()],
+  [assert.strictEqual, `${os.release}`, os.release()],
+  [assert.strictEqual, `${os.type}`, os.type()],
+  [assert.strictEqual, `${os.endianness}`, os.endianness()]
+].forEach((set) => {
+  set[0](set[1], set[2], `${set[1]}, ${set[2]}`);
```

Comment:
I’m not actually sure whether the extra error message argument here is helpful, and maybe I would just inline `assert.strictEqual` here instead of passing it as part of `set`. (Just a suggestion, feel free to ignore me if you disagree)